modules/nw-nwop-modifying-config-running.adoc

```diff
@@ -0,0 +1,75 @@
+// Module filename: nw-nwop-modifying-config-running.adoc.adoc
+// Module included in the following assemblies:
+// * networking/configuring-network-operator.adoc
+
+[id='nw-nwop-modifying-config-running-{context}']
+= Modifying Network Configuration in a Running Cluster
+
+You can only modify `kubeProxy` configuration parameters in
+a running cluster.
+
+The Network Operator reconciles the state of the cluster
+against a desired configuration.
+
+.Prerequisites
+
+* Login to a running cluster.
+
+.Procedure
+
+. Use the following command to edit the
+`NetworkConfig.networkoperator.openshift.io` Custom Resource (CR).
++
+----
+$ oc edit networkconfig
+----
++
+The example output below shows a `NetworkConfig.networkoperator.openshift.io` CR
+that has been configured with default values.
++
+[source,yaml]
+----
+  # Please edit the object below. Lines beginning with a '#' will be ignored,
+  # and an empty file will abort the edit. If an error occurs while saving this file will be
+  # reopened with the relevant failures.
+  #
+  apiVersion: networkoperator.openshift.io/v1
+  kind: NetworkConfig
+  metadata:
+    creationTimestamp: 2019-02-05T10:30:33Z
+    generation: 1
+    name: default
+    namespace: ""
+    resourceVersion: "591"
+    selfLink: /apis/networkoperator.openshift.io/v1/networkconfigs/default
+    uid: xxxxxx
+  spec:
+    clusterNetworks:
+    - cidr: 10.128.0.0/14
+      hostSubnetLength: 9
+    defaultNetwork:
+      type: OpenShiftSDN
+    serviceNetwork: 172.30.0.0/16
+  status: {}
+
+----
+
+. Add the following code to the `spec` field in the
+`NetworkConfig.networkoperator.openshift.io` CR.
++
+[source,yaml]
+----
+spec:
```

Comment:
Make sure that the new assembly contains sufficient information about the parameter values.